egs/wsj/s5/utils/lang/compute_sentence_probs_arpa.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python
+
+# Dongji Gao
+
+# We're using python 3.x style but want it to work in python 2.x
+
+from __future__ import print_function
+import argparse
+import sys
+
+parser = argparse.ArgumentParser(description="This script evaluates the lob probabilty (10 based) of each sentence "
```

Comment:
This should be 'log probability'.
And wouldn't it be more convenient for it to default to outputting log base e (like Kaldi uses) and maye we can add an option to output log base 10 if it's needed in the future?

Reply:
The log base in arpa is 10, so I directly used it. I will fix it and add a log base option.